components/content-search/index.tsx

```diff
@@ -1,13 +1,20 @@
 import { Spinner, NavigableMenu, Button, SearchControl } from '@wordpress/components';
-import apiFetch from '@wordpress/api-fetch';
-import { useState, useRef, useEffect, useCallback } from '@wordpress/element';
-import { addQueryArgs } from '@wordpress/url';
+import { useState, useRef } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import styled from '@emotion/styled';
-import SearchItem, { Suggestion } from './SearchItem';
+import {useMergeRefs} from '@wordpress/compose';
```

Comment:
[nit] space between brackets

Reply:
Something is wrong with the eslint setup... I think we need to switch over to `@typescript-eslint/parser`

Reply:
Fixed the linter